internal/config/config.go

```diff
@@ -40,8 +40,9 @@ type Config struct {
 	// MetricsNamespace defines the namespace that will be used for the prometheus metrics.
 	MetricsNamespace string `json:"metricsNamespace"`
 	ProbeAddress     string `json:"probeAddress"`
-	// OrganizationID is the snyk organization ID where data should be routed to.
-	OrganizationID string `json:"organizationID"`
+
+	// Routes contain configuration resources from which namespaces are routed for which organization
```

Comment:
```suggestion
	// Routes define which resources should be sent to which organizations.
```
(as the routes also work on non-namespaced resources)